Fixes
    ../sysdeps/i386/i686/memcmp.S: Assembler messages:
    ../sysdeps/i386/i686/memcmp.S:390: Error: can't resolve  `.rodata' {.rodata section} - `memcmp' {.text section}
when building on macosx.

http://sourceware.org/cgi-bin/cvsweb.cgi/libc/sysdeps/i386/i686/memcmp.S?cvsroot=glibc
Revision 1.4, Thu Apr 28 18:19:25 2005 UTC by drepper
Branch: MAIN
CVS Tags: fedora-glibc-20050503T0852
Changes since 1.3: +2 -2 lines
Diff to previous 1.3 (colored)

Move misplaced END.

http://sourceware.org/cgi-bin/cvsweb.cgi/libc/sysdeps/i386/i686/memcmp.S.diff?r1=1.3&r2=1.4&cvsroot=glibc

===================================================================
RCS file: /cvs/glibc/libc/sysdeps/i386/i686/memcmp.S,v
retrieving revision 1.3
retrieving revision 1.4
diff -u -r1.3 -r1.4
--- libc/sysdeps/i386/i686/memcmp.S	2004/04/01 23:28:37	1.3
+++ libc/sysdeps/i386/i686/memcmp.S	2005/04/28 18:19:25	1.4
@@ -1,5 +1,5 @@
 /* Compare two memory blocks for differences in the first COUNT bytes.
-   Copyright (C) 2004 Free Software Foundation, Inc.
+   Copyright (C) 2004, 2005 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
@@ -350,6 +350,7 @@
 	sbbl	$-1, %eax
 	popl	%esi
 	RETURN
+END (BP_SYM (memcmp))
 
 	.section	.rodata
 	ALIGN (2)
@@ -387,7 +388,6 @@
 	.long	L(30bytes) - . + 0x78
 	.long	L(31bytes) - . + 0x7c
 
-END (BP_SYM (memcmp))
 
 #undef bcmp
 weak_alias (BP_SYM (memcmp), BP_SYM (bcmp))
